core/rovecomm.py

```diff
@@ -381,13 +391,17 @@ def write(self, packet):
             for i in packet.data:
                 rovecomm_packet = rovecomm_packet + struct.pack('>' + packet.data_type, i)
 
-            # establish a new connection if the destination has not yet been connected to yet
-            if self.connect(packet.ip_address) == 0:
-                return 0
+            for address in self.open_sockets:
+                self.open_sockets[address].send(rovecomm_packet)
```

Comment:
We should split off open connections we initiate and open connections that were requested from us, and then only send packets to people who requested from us. That was we don't do something like send logs to drive board.